samples/04_gis_analysts_data_scientists/classifying_human_activity_using _tabPFN_classifier.ipynb

```diff
@@ -0,0 +1,2179 @@
+{
```

Comment:
<p>In the training dataframe above we use &nbsp;<code>Activity</code>&nbsp;as the target label to be predicted, using the rest of the features as explanatory variables&nbsp;<code>X</code>. We define the explanatory variables as follows:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2189/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>